backend/webapi/Features/ClientLogs/Create.cs

```diff
@@ -0,0 +1,54 @@
+namespace Pidp.Features.ClientLogs;
+
+using FluentValidation;
+using Pidp.Data;
+using Pidp.Models;
+
+public class Create
+{
+    public class Command : ICommand
+    {
+        public int Id { get; set; }
```

Comment:
Create `Command`s will almost never include the `Id`; we don't let the front-end decide the Key of the row we create for many reasons, we let the database generate it.